pentaho-worker-nodes-kafka/pom.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>pentaho</groupId>
+    <artifactId>pentaho-osgi-bundles</artifactId>
+    <version>9.1.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>pentaho-worker-nodes-kafka</artifactId>
```

Comment:
Why to create a new bundle? Can't this just be added in the bundle created by the DFM team for spring-kafka purposes? 

Is there anything that ties it to WNs logic that makes it need to have worker-nodes in the name? The plugins we made don't mention worker-nodes for the reason that they can be reused by other teams, I'd say the same logic applies here

Reply:
Yeah, moving it into the DFM team's module does make sense. I've moved the code over. 
I cleaned up the git history and force pushed to avoid unreadable diffs.

I'm currently testing that change/ironing out some issues -- switching this to a draft PR for the time being.